library/attachment-viewer/src/main/java/im/vector/lib/attachmentviewer/AnimatedImageViewHolder.kt

```diff
@@ -20,7 +20,7 @@ import android.view.View
 import im.vector.lib.attachmentviewer.databinding.ItemAnimatedImageAttachmentBinding
 
 class AnimatedImageViewHolder constructor(itemView: View) :
-        BaseViewHolder(itemView) {
+    BaseViewHolder(itemView) {
```

Comment:
is it the "continuation indent" thing?
I do not like the new format :/

```kotlin
class AnimatedImageViewHolder constructor(itemView: View) :
    BaseViewHolder(itemView) {
    val views = ItemAnimatedImageAttachmentBinding.bind(itemView)
```

is more confusing than

```kotlin
class AnimatedImageViewHolder constructor(itemView: View) :
        BaseViewHolder(itemView) {
    val views = ItemAnimatedImageAttachmentBinding.bind(itemView)
```
if I am reading it correctly.